### DISCUSSION TITLE
Compiling source code

### DISCUSSION THREAD

**Discussion by WilsonBradley** [curif/AgeOfJoy-2022.1#69] [General]
What version compiler (link please) is used to compile the source code?
I would like to start compiling to test.
Example;
https://github.com/curif/AgeOfJoy-2022.1/tree/0.2.0

**Comment by Hoongming** [comment]
hi,

I am also interested in this. I tried my hand at this, installed Unity and Visual Studio Code, read a few tutorials but havent been able to understand where the build script comes in. Looks like it might be a Visual Studio Community edition project so installing that instead.

Would be good if there was a guide on how your projects are setup to compile. Know that there are countless numbers of enthusiasts but some come from beginner backgrounds.

Thanks,

Johnny

**Reply by curif** [reply]
Hey. Yes I know it's very hard to understand. I think you could get some help in the discord channel: https://discord.gg/b83ykCM9Xp as there isn't any docs about Unity yet.

Regards

**Reply by WilsonBradley** [reply]
> hi,
> 
> I am also interested in this. I tried my hand at this, installed Unity and Visual Studio Code, read a few tutorials but havent been able to understand where the build script comes in. Looks like it might be a Visual Studio Community edition project so installing that instead.
> 
> Would be good if there was a guide on how your projects are setup to compile. Know that there are countless numbers of enthusiasts but some come from beginner backgrounds.
> 
> Thanks,
> 
> Johnny

Agree..
